agent/statemanager/state_manager.go

```diff
@@ -60,11 +60,14 @@ const (
 	// 9) Add 'ipToTask' map to state file
 	// 10) Add 'healthCheckType' field in 'apicontainer.Container'
 	// 11)
-	//  a) Add 'PrivateDNSName' field to 'api.ENI'
-	//  b)Remove `AppliedStatus` field form 'apicontainer.Container'
+	//   a) Add 'PrivateDNSName' field to 'api.ENI'
+	//   b)Remove `AppliedStatus` field form 'apicontainer.Container'
 	// 12) Deprecate 'TransitionDependencySet' and add new 'TransitionDependenciesMap' in 'apicontainer.Container'
 	// 13) Add 'resources' field to 'api.task.task'
-	ECSDataVersion = 13
+	// 14)
+	//   a) Add 'V3EndpointID' field to 'Container' struct
+	//   b) Add 'v3EndpointIDToTask' and 'v3EndpointIDToContainerName' maps to engine state
```

Comment:
Are you sure `v3EndpointIDToContainerName` and `v3EndpointIDToTask` are saved in the state file, can you verify that?

Reply:
yes i had a manual test and verified that